### ISSUE TITLE
The inconsistent performance on COLLAB dataset

### ISSUE THREAD

**Issue by ZhuYun97** [THUDM/GraphMAE#16]
Thanks for your fabulous codes, but I meet some problems while reproducing. After running 5 runs(`sh scripts/run_graph.sh COLLAB 0` using seed: 0,1,2,3,4), the performance on COLLAB dataset is 78.78±0.46 which remains a large margin with the reported value(80.32±0.46). 
I check the model configuration in the Appendix and find the hidden size of GIN is 512(but 256 in the [config.yaml](https://github.com/THUDM/GraphMAE/blob/1be594160bb7c1fde0d16ced6af1e88b53612e49/configs.yml#L255)). After changing the hidden size to 512, the performance(78.70±0.32) is still inferior to the paper. 
Is there anything I missing? hope for your reply.

**Comment by THINK2TRY** [comment]
@ZhuYun97 Thanks for your interest. The scaling coefficient `alpha_l` should be set to 1 in COLLAB, while it is 2 by default now. Sorry to miss this hyper-parameter in configs.yml. We have updated `configs.yml`. Hope this help.

**Comment by ZhuYun97** [comment]
Thanks for your instant reply.

**Comment by ZhuYun97** [comment]
Sorry to bother you again. The performance on MUTAG dataset is 86.47±1.81, but I find that all the hyperparameters are the same with Appendix. Is there any other things I should change?

**Comment by ZhuYun97** [comment]
I use the latest `config.yml`. And I run on NVIDIA 3090(I also test on NVIDIA 1080Ti, but get similar results).
```
MUTAG:
  num_hidden: 32
  num_layers: 5
  lr: 0.0005
  weight_decay: 0.00
  mask_rate: 0.75
  drop_edge_rate: 0.0
  max_epoch: 20
  encoder: gin
  decoder: gin
  activation: prelu
  loss_fn: sce
  scheduler: False  
  pooling: sum
  batch_size: 64
  alpha_l: 2
  replace_rate: 0.1
  norm: batchnorm
  in_drop: 0.2
  attn_drop: 0.1
  alpha_l: 2
```

**Comment by ZhuYun97** [comment]
There are  logs.

------ Use best configs ------
Namespace(activation='prelu', alpha_l=2, attn_drop=0.1, batch_size=64, concat_hidden=False, dataset='MUTAG', decoder='gin', deg4feat=False, device=0, drop_edge_rate=0.0, encoder='gin', fewshot=False, fewshot_num=10, finetune=False, in_drop=0.2, linear_prob=True, load_model=False, logging=False, loss_fn='sce', lr=0.0005, lr_f=0.001, mask_rate=0.75, max_epoch=20, max_epoch_f=0, negative_slope=0.2, norm='batchnorm', num_heads=2, num_hidden=32, num_layers=5, num_out_heads=1, optimizer='adam', pooling='sum', replace_rate=0.1, residual=False, save_model=False, scheduler=False, seeds=[0, 1, 2, 3, 4], use_cfg=True, warmup_steps=-1, weight_decay=0.0, weight_decay_f=0.0)
Use node label as node features
******** # Num Graphs: 188, # Num Feat: 7, # Num Classes: 2 ********
####### Run 0 for seed 0
Epoch 19 | train_loss: 0.1787: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:01<00:00, 11.44it/s]
#Test_f1: 0.8348±0.0845
####### Run 1 for seed 1
Epoch 19 | train_loss: 0.2188: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:01<00:00, 15.42it/s]
#Test_f1: 0.8512±0.0526
####### Run 2 for seed 2
Epoch 19 | train_loss: 0.2620: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:01<00:00, 14.54it/s]
#Test_f1: 0.8883±0.0367
####### Run 3 for seed 3
Epoch 19 | train_loss: 0.2816: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:01<00:00, 14.29it/s]#Test_f1: 0.8830±0.0457
####### Run 4 for seed 4
Epoch 19 | train_loss: 0.2814: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 20/20 [00:01<00:00, 13.03it/s]
#Test_f1: 0.8830±0.0697
#final_acc: 0.8681±0.0212

**Comment by ZhuYun97** [comment]
After I change the device to NIVIDIA 2080 Ti, the performance gets `88.29±1.22`. I guess the MUTAG dataset is too small, so the experiments are not so stable.

**Comment by THINK2TRY** [comment]
Indeed, MUTAG is too small, so the variance is larger and the results are not as stable as other results. And previously, we also found that some randomness can still exist in NIVIDIA 3090 Ti even when we use the same random seed. So you can just run the experiments on 2080 Ti.
